FDD_RF_Module/FDD_RF_Modeling.py

```diff
@@ -86,10 +88,18 @@ def inputs_output_generator(self, train_or_test):
 
             for simulation_data_file_name in simulation_data_file_list:
                 temp_raw_FDD_data = pd.read_csv(f'data\\{self.weather}\\{self.weather}\\{simulation_data_file_name}')
+                #temp_raw_FDD_data = temp_raw_FDD_data.apply(pd.to_numeric, errors='coerce')
+                #temp_raw_FDD_data = temp_raw_FDD_data.dropna()
                 temp_raw_FDD_data = temp_raw_FDD_data.groupby(temp_raw_FDD_data.index // (self.aggregate_n_runs)).mean().iloc[:,0:-8]
                 temp_raw_FDD_data['label'] = meta_data.loc[meta_data.id == simulation_data_file_name[0:-12]].fault_type.values[0]
-                fault_inputs_output = pd.concat([fault_inputs_output, temp_raw_FDD_data], axis = 0)
-
+                # Splitting training and testing data
+                temp_raw_FDD_data_train, temp_raw_FDD_data_test = train_test_split(temp_raw_FDD_data, test_size=0.2, random_state=np.random.RandomState(self.randomseed))
```

Comment:
`random_state=np.random.RandomState(self.randomseed)` make sure the reproducibility of random split.

Reply:
ahh did not know there is an easy method for splitting data in sklearn package!